tests/foreman/ui/test_host.py

```diff
@@ -17,46 +18,72 @@ def test_create_host(self):
         @Assert: Host is created
 
         """
+        host = entities.Host()
+        host.create_missing()
+        os_name = u'{} {}'.format(
```

Comment:
@abalakh Yes, I was trying to point out that `str()` is unnecessary in that context.

By the way, we use `{0} {1}` because Python 2.6 and earlier require it. Maintaining a list of explicit indices is a hassle, so I avoid adding indices when working with code that only dependson Python 2.7 and above.